File: tools/conf/output/ts/HelpConfig.ts
class HelpConfig {
    private static _dic:Dictionary<HelpConfig> = new Dictionary<HelpConfig>();
    private static _arr:Array<HelpConfig> = new Array<HelpConfig>();

	 public Id :number ;
     public HelpText :string ;
    

    
    public constructor(){

    }


    public static get Dic():Dictionary<HelpConfig>{
        return  HelpConfig._dic;
    }
    public static get Arr():Array<HelpConfig>{
        return HelpConfig._arr;
    }
    public static parse(json:any[]){

        for(var i = 0; i < json.length; i++){
            let config:HelpConfig = <HelpConfig>json[i];
            HelpConfig._arr.push(config);
            HelpConfig._dic.add(config.Id,config);
        }
    }
}

File: tools/conf/output/go/HelpConfig.go
package conf

import (
	//"bufio"
	"encoding/json"
	"fmt"
	//"io"
	"io/ioutil"
	"os"
	//"strings"
)

type HelpConfigElem struct {
	Id int
	HelpText string
	
}

type HelpConfig0 struct {
	Dic map[int]*HelpConfigElem
	Arr []*HelpConfigElem
}

var (
	HelpConfig HelpConfig0
)

func loadHelpConfig(path string) {
	file, err := os.Open(path)
	defer func() {
		if file != nil {
			file.Close()
		}
	}()
	if err != nil {
		panic(fmt.Sprint("load config error, path=", path))
	}

	HelpConfig.Dic = make(map[int]*HelpConfigElem)
	HelpConfig.Arr = make([]*HelpConfigElem, 0, 32)

	bt,err  :=  ioutil.ReadAll(file)
	json.Unmarshal(bt, &HelpConfig.Arr)
	for _,conf := range HelpConfig.Arr{
		HelpConfig.Dic[conf.Id] = conf
	}
	
}
